### PR TITLE
bump `zombienet` version to v1.3.37

### PR DESCRIPTION
**Pull request by pepoviola**
Bump zombienet version and removing the `is up` assertions since that is now checked by zombienet before running the tests.

Thanks!

### CODE REVIEW

**Review by michalkucharczyk — APPROVED** [review]

**Review by sergejparity — APPROVED** [review]

**Review by melekes — APPROVED** [review]
👍

**Review by alvicsam — APPROVED** [review]

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -55,7 +55,7 @@ variables:
   RUSTY_CACHIER_COMPRESSION_METHOD: zstd
   NEXTEST_FAILURE_OUTPUT: immediate-final
   NEXTEST_SUCCESS_OUTPUT: final
-  ZOMBIENET_IMAGE: "docker.io/paritytech/zombienet:v1.3.34"
+  ZOMBIENET_IMAGE: "docker.io/paritytech/zombienet:v1.3.35"
```
Comment (pepoviola) [inline .gitlab-ci.yml:58]:
```suggestion
  ZOMBIENET_IMAGE: "docker.io/paritytech/zombienet:v1.3.37"
```

**Comment by bkchr** [comment]
@pepoviola can we merge?

**Comment by pepoviola** [comment]
> @pepoviola can we merge? 

Yes, thanks!

**Pull request merged**